### ISSUE TITLE
UnexpectedParameter errors when using KinesisStreamSpecification

### ISSUE THREAD

**Issue by MrNoyce** [raisenational/serverless-dynamodb#9]
Hello,

When using Kinesis Data Streams with DynanoDB, the above error is thrown once dynamoDB local is started.

Similar to the fix in https://github.com/raisenational/serverless-dynamodb/pull/1, the below code fixes the code locally for me.
```
// line 370 index.js
            if(migration.KinesisStreamSpecification) {
                delete migration.KinesisStreamSpecification;
            }
```

Could you please look into this I have just moved over to this package and this is a bit of a blocker for me :).

Many thanks,
MrNoyce

**Comment by domdomegg** [comment]
Sure, this definitely sounds like something we can support.

Generally it's probably worth (for us as a library) going through https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html and double checking everything is being removed that shouldn't be passed to the table creation API (see #11).

**Comment by domdomegg** [comment]
Fixed in v0.2.43

**Comment by MrNoyce** [comment]
Hello @domdomegg,

Thank you for the quick turn around and support on this issue. Thank you also for the tip as the KinesisSpecification property is conditionally used inside my CF template and maybe something isn't being cleaned up / ignored correctly.
